apps/site/components/Downloads/Release/ChangelogLink.tsx

```diff
@@ -4,19 +4,17 @@ import type { FC, PropsWithChildren } from 'react';
 import { useContext } from 'react';
 
 import LinkWithArrow from '@/components/Downloads/Release/LinkWithArrow';
+import Link from '@/components/Link';
 import { ReleaseContext } from '@/providers/releaseProvider';
 
+const BASE_CHANGELOG_URL = 'https://github.com/nodejs/node/releases/tag/v';
```

Comment:
move this to next.constants.mjs

Reply:
addressed via https://github.com/nodejs/nodejs.org/pull/7155/commits/5fbe3f92800b6f283f07c51ee1c5a23dec71f6be